app/config.yml

```diff
@@ -0,0 +1,4 @@
+theme_search_paths:
+  -
+  - modules/courses
+  - modules
```

Comment:
you probably want to also define `modules_that_allow_delete_on_deploy:` with all the modules, as currently if you delete a file from your filesystem and deploy the code to the instance, it will not be deleted from there